lib/python/Components/UsageConfig.py

```diff
@@ -1274,7 +1274,7 @@ def upgrade(configItem, name, valuemap = None, mapper = None):
 		print "[UsageConfig] Upgrading EPG settings"
 
 		okMap = {"Zap + Exit": "zapExit", "Zap": "zap"}
-		infoMap = {"Channel Info": "openEventView", "Single EPG": "openSingleEPG"}
+		infoMap = {"Event Info": "openEventView", "Single EPG": "openSingleEPG"}
```

Comment:
Don't think this change is actually correct. This should be the old value, which used to _Channel Info_

Reply:
OK, I'll change it. Or you can just make the other change your end and close this request

Reply:
Changed my end.